CMakeLists.txt

```diff
@@ -71,6 +71,12 @@ if(CMAKE_CXX_COMPILER_ID MATCHES Clang)
     endif()
 endif()
 
+if(CMAKE_SYSTEM_PROCESSOR STREQUAL x86_64 AND CMAKE_SIZEOF_VOID_P EQUAL 4)
```

Comment:
`x86_64`?

Reply:
Maybe it's different on true 386, but I have not way of checking.

Reply:
I'm sure it should be `i386` on any non-64-bit x86 (at some point they tried to retrofit `ia32` though).

Reply:
Some search suggest that it may return the following: `x86`, `i386`, `i686`.

See https://boringssl.googlesource.com/boringssl/+/refs/heads/2272/CMakeLists.txt and http://www.autoscool-clermont.com/blum/wp-content/themes/matrix/inc/kirki/assets/js/vendor/codemirror/mode/cmake/index.html (which has `set(X86_ALIASES x86 i386 i686 x86_64 amd64)`).